modules/network-observability-operator-install.adoc

```diff
@@ -17,7 +21,7 @@ You can install the Network Observability Operator using the {product-title} web
 +
 * *spec.agent.ebpf.Sampling* : Specify a sampling size for flows. Lower sampling sizes will have higher impact on resource utilization. For more information, see the `FlowCollector` API reference, under spec.agent.ebpf. 
 * *spec.deploymentModel*: If you are using Kafka, verify Kafka is selected.
-* *Loki.url*: Since authentication is specified separately, this URL needs to be updated to `https://loki-gateway.http-netobserv.svc:8080/api/logs/v1/network`
+* *Loki.url*: Since authentication is specified separately, this URL needs to be updated to `https://loki.gateway.http.netobserv.svc:8080/api/logs/v1/network`
```

Comment:
it's `https://loki-gateway-http.netobserv.svc:8080/api/logs/v1/network`

Reply:
Ok. I think I've got it correct now.